test-currentRanges.cpp

```diff
@@ -0,0 +1,26 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "test/catch.hpp"
+#include "CurrentRanges.h"
+#include "stdlib.h"
+#include "math.h"
+// Number of reading betweem 3 to 5 is 4 so testcase shall pass if this is in permitted range
+
+//PASSING TESTCASE
+TEST_CASE("To check if the samples are in range") {
```

Comment:
Whenever you describe a test with the word 'check', it means something is yet to be specified.
Try mentioning the inputs and expected outputs in the name of the test.
E.g., does 'in range' include the limits or not? That should be evident in the test, triggering clarification with the consumers of the code.